hackathon_site/hardware/views.py

```diff
@@ -82,6 +82,16 @@ def post(self, request, *args, **kwargs):
         return self.create(request, *args, **kwargs)
 
 
+class IncidentDetailView(mixins.RetrieveModelMixin, generics.GenericAPIView):
+    queryset = Incident.objects.all()
+
+    serializer_class = IncidentListSerializer
```

Comment:
why are you using the incident list serializer for a detail view?

Reply:
I believe it still worked, do you think I should create a new serializer for the detail view?